docs/geo_upload.md

```diff
@@ -7,7 +7,63 @@ Create metadata submission spreadsheet
 ```
 python geo_metadata.py -m local -d <Dataset_accession>
 ```
-	This will create three files
-	- \<Dataset_accession>\_metadata.csv: This is a csv of all the study, sample, protocol, and fasta file metadata.
-	- \<Dataset_accession>\_md5sum.csv: This is a csv of all the fasta and raw matrix file names and checksums.
-	- \<Dataset_accession>\_s3_uri.csv: This is a list of the s3 URI for all fastq and raw matrix files.
+- This will create three files:
+	- <Dataset_accession>_metadata.csv: This is a csv of all the study, sample, protocol, and fasta file metadata.
+	- <Dataset_accession>_md5sum.csv: This is a csv of all the fasta and raw matrix file names and checksums.
+	- <Dataset_accession>_s3_uri.csv: This is a list of the s3 URI for all fastq and raw matrix files.
+- Upload the metadata and md5sum csv files in a google spreadsheet. Have wrangler add any additional contributor or protocol information.
+
+Launch EC2
+----------------
+- Launch an EC2 instance with enough disk space for 2 upload processes running in parallel.
+- Install c compiler:
+```
+sudo yum install gcc
+```
+- Install ncftp on machine:
+```
```

Comment:
I would either break this into separate blocks so you can copy/paste each line separately, or combine them into 1 line with `;` separating each line, so you copy/paste the whole thing